pkg/util/os/limits.go

```diff
@@ -0,0 +1,42 @@
+package os
+
+import (
+	"fmt"
+	"log"
+
+	"golang.org/x/sys/unix"
+)
+
+func setFileLimit(n uint64) error {
+	limit := unix.Rlimit{
+		Cur: n,
+		Max: n,
+	}
+
+	log.Printf("setting limit to %d", n)
```

Comment:
Please use our logger (`core.Log`) instead of `"log"`

Reply:
fixed